torchci/lib/JobClassifierUtil.ts

```diff
@@ -0,0 +1,196 @@
+import { GroupData, RowData } from "./types";
+
+const groups = [
```

Comment:
I took this from the old HUD just to keep parity between the two. I think changing the names is a good idea since some of the workflows and stuff have different naming standards like some of the builds vs publish to s3 or something. It might also allow for better classification/parsing and stuff like how you described but it seems a bit out of scope.

Reply:
Just made it
https://github.com/pytorch/pytorch/issues/72814


> One more followup: right now the logic for collapsing groups is wrong; if I have multiple groups open and collapse one, they will all collapse. Clicking to collapse a group should only collapse that group.